Taggers/python/flashggTags_cff.py

```diff
@@ -63,16 +63,20 @@
                                        rhoTag = cms.InputTag('fixedGridRhoFastjetAll'),
 #                                       tthMVAweightfile = cms.FileInPath("flashgg/Taggers/data/TMVAClassification_tth_hadronic_2017Data_35vars_v0.weights.xml"),
                                        tthMVAweightfile = cms.FileInPath("flashgg/Taggers/data/TMVAClassification_tth_hadronic_2017Data_30vars_v0.weights.xml"),
+				       topTaggerXMLfile = cms.FileInPath("flashgg/Taggers/data/resTop_xgb_csv_order_deepCTag.xml"),
```

Comment:
Are these trainings year dependent? If so we should steer the file name from the metaconditions.

Reply:
Currently, no -- there is a single training for all 3 years. I think it is likely that we will keep it as a single training for all 3 years, and add the year information as an input to the BDTs.